libs/vm/src/module.cpp

```diff
@@ -143,72 +149,122 @@ double toFloat64(VM * /* vm */, AnyPrimitive const &from)
   return Cast<double>(from);
 }
 
+class AllClassInterfaces
+{
+private:
+  template <typename T>
+  using ClassInterface = Module::ClassInterface<T>;
+
+public:
+  ClassInterface<Address> address;
+  ClassInterface<IArray>  iarray;
+  ClassInterface<IMap>    imap;
+  ClassInterface<IMatrix> imatrix;
+  ClassInterface<IState>  istate;
+  ClassInterface<String>  string;
+};
+
+void InstantiateTypes(AllClassInterfaces &interfaces)
```

Comment:
Splitting aspects of type setup from *single* place in to multiple new functions obscures the view on how the type is actually setup.
Before, the type was fully setup on signle place, where developer can clearly see all spects without necessity to dig around to get the full picture.

Reply:
See my answer above.